src/chains/models.py

```diff
@@ -42,21 +42,33 @@ class Chain(models.Model):
         default="#000000",
         help_text="Please use the following format: <em>#RRGGBB</em>.",
     )
+    ens_registry_address = EthereumAddressField(null=True, blank=True)
     gas_price_oracle_url = models.URLField(blank=True, null=True)
     gas_price_oracle_parameter = models.CharField(blank=True, null=True, max_length=255)
-    ens_registry_address = EthereumAddressField(null=True, blank=True)
     gas_price_oracle_gwei_factor = models.DecimalField(
         default=1,
         max_digits=19,
         decimal_places=9,
         verbose_name="Gwei multiplier factor",
         help_text="Factor required to reach the Gwei unit",
     )
+    gas_price_fixed_wei = Uint256Field(
+        verbose_name="Fixed gas price (wei)", blank=True, null=True
+    )
```

Comment:
This is output as a string as you can see in the tests, right?

Reply:
Django generates a human readable string for each field by default (to show in the admin interface).

So `gas_price_oracle_gwei_factor` would be shown as `Gas Price Oracle Gwei Factor` (in the admin)
For `gas_price_fixed_wei` it could be confusing having it generated as: `Gas Price Fixed Wei` so `verbose_name` just overrides that default.